content/events/IPFS-friends-ethcc.md

```diff
@@ -0,0 +1,26 @@
+---
+eventName: "IPFS + Friends Cafe: Paris" #the name of your track or event and its mandatory
+
+eventDescription: "​Cafes emerged in Europe in the 17th century, serving as a place to foster community and promote the exchange of ideas and philosophy. Paris also welcomed in this Age of Englightenment.
+
+​Fitting for us, we'd like to invite you to join us in Paris during EthCC at the IPFS & Friends Cafe to discuss or present anything IPFS or Dweb related.
+
+Our venue is not yet confirmed but we are looking at July 17." #short description of your track or event limiting to 100-150 characters
+
+location: "Paris, France" or "Location TBD" #Specify the location of the event.If you aren't sure about the location then mention "Location TBD"
+
+website: "http://lu.ma/ipfsnfriends-paris23" #make sure to have all the relevant information: dates, venue, program, ticketing (if any), etc. or just leave it blank
+
+startDate: "07/17/2023" # the start date of the event , date format is MM/DD/YYYY eg: if it is February 16th 2023 => 02/16/2023
+
+endDate: "07/17/2023" #the end date of the event , date format is MM/DD/YYYY eg: if it is February 18th 2023 => 02/18/2023
+
+tag: "PLN Event" #Mention any one of the event types. Don't leave it blank.
+
+dateTBD: true #If you don't have concrete dates, add the approximate dates & set dateTBD: true.
+
+#Event host names and their respective logos(preferred size is 48px width, 48px height)-place the logo file on the path 'public/uploads' for eg.   - IPFS|ipfs-logo.png
+
+eventHosts:
+  - IPFS|<host_logo>
```

Comment:
@laeccles There is another event that has the same event host 'IPFS' but with a different logo - 

<img width="43" alt="image" src="https://github.com/memser-spaceport/pln-events/assets/120092872/cf575a4c-f583-4694-8034-e80af419f423">

Please confirm if the above logo can be used (since the event host is IPFS).